### ISSUE TITLE
How to use your project for image inpaint training

### ISSUE THREAD

**Issue by create-li** [LS4GAN/uvcgan2#41]
Hi @mjz0110,

That depends on which model you would like to train:
1. If you would like to pre-train an image inpainting model, then simply run the pre-training on a machine with several GPUs and set CUDA_VISIBLE_DEVICES environment variable to "comma separated list of GPU indices" . The pre-training script will automatically use data parallel training.
2. If you would like to train a style transfer model, then unfortunately, there is currently no way to train it on multiple GPUs.

_Originally posted by @usert5432 in https://github.com/LS4GAN/uvcgan2/issues/38#issuecomment-2292448227_

**Comment by create-li** [comment]
Hello, thank you for your outstanding contribution. How to train the inpaint model and which script and configuration to use in the project. However, your project only has image translation training scripts.

**Comment by usert5432** [comment]
Hello @create-li,

Thank you for your interest in our work. We provide the following scripts to train simple image masked image inpainting models:
```
scripts/afhq/pretrain_afhq.py
scripts/anime2selfie/pretrain_anime2selfie.py
scripts/celeba/pretrain_celeba.py
scripts/celeba_hq/pretrain_celebahq.py
```

We tried to describe how to run them [here](https://github.com/LS4GAN/uvcgan2?tab=readme-ov-file#3-generator-pre-training).
